src/game_api/rpc.cpp

```diff
@@ -232,7 +218,7 @@ uint32_t get_entity_flags(uint32_t uid)
 
 void set_entity_flags(uint32_t uid, uint32_t flags)
 {
-    if (uid == 0)
+    if (uid < 1)

```

Comment:
Why make this change?

Reply:
i didn't saw it's already unsigned xd 
i was actually thinking do we even need that check? if you give it wrong uid it will just not find it right?